### DISCUSSION TITLE
Daily challange leaderboard

### DISCUSSION THREAD

**Discussion by Adhm1234** [ppy/osu#29535] [Ideas]
Hello!

Small suggestion that could make Daily challange more engaging for the long term by simply creating a leaderboard for amount of daily streaks. Obviously a lot of people currently may still have the maximum daily streak going thus i suggest putting the person with most top 10%'s on the number 1 spot or something along the lines of this.

**Comment by Tenexxt** [comment]
Already anwsered in #28694
Could be done if there is enough intrest in a streak leaderboard, so the more people will ask for this the higher the chance this gets added.
